source/_integrations/tami4.markdown

```diff
@@ -18,6 +18,7 @@ ha_integration_type: integration
 Support for [Tami4 Edge / Edge+ by Strauss](https://www.tami4.co.il/tami4edge-collection) in Home Assistant.
 
 You can boil water and get filter / UV information, such as when you need to replace them.
+Also, all of your personal drinks will be available as buttons for you to use.
```

Comment:
Below the config flow, please add a section called ## Entities, ### Buttons, listing the names and descriptions of all available buttons. The same would be true for ### Sensors, but those are outside the scope of this PR.